src/main/java/gregtech/client/utils/ToolChargeBarRenderer.java

```diff
@@ -96,9 +95,12 @@ private static void overpaintVanillaRenderBug(BufferBuilder worldrenderer, int x
 
     public static void renderBarsTool(IGTTool tool, ItemStack stack, int xPosition, int yPosition) {
         boolean renderedDurability = false;
-        NBTTagCompound tag = GTUtility.getOrCreateNbtCompound(stack);
-        if (!tag.getBoolean(ToolHelper.UNBREAKABLE_KEY)) {
-            renderedDurability = renderDurabilityBar(stack.getItem().getDurabilityForDisplay(stack), xPosition, yPosition);
+        NBTTagCompound tag = stack.getTagCompound();
+        if (tag == null || !tag.getBoolean(ToolHelper.UNBREAKABLE_KEY)) {
+            double durability = stack.getItem().getDurabilityForDisplay(stack);
+            if (durability != 0.0) {
+                renderedDurability = renderDurabilityBar(durability, xPosition, yPosition);
```

Comment:
For now `renderDurabilityBar` just returns `true` so setting `renderedDurability` flag could be simplified to just storing `true`.

Or, even better, inline the `renderDurabilityBar` method, because this is the only usage.

Also wondering if `durability > 0` would be better than `durability != 0`.